<?xml version="1.0"?>
<launch>
    <include file="$(find audio_capture)/launch/capture.launch" >
        <param name="format" value="wave" />
    </include>

    <node pkg="snowboy_ros" type="hotword_detector_node" name="snowboy">
        <param name="resource_filename" value="$(find snowboy_ros)/resources/common.res"/>
        <param name="model_filename" value="$(find snowboy_ros)/resources/Hey_Amigo.pmdl"/>
        <param name="sensitivity_str" value="0.01"/>
        <param name="audio_gain" value="1.0"/>
    </node>
</launch>
